tests/crictl/crictl_test.go

```diff
@@ -214,7 +214,7 @@ func TestCrictlSptRumprunRedis(t *testing.T) {
 }
 
 func TestCrictlQemuUnikraftRedis(t *testing.T) {
-	containerImage := "harbor.nbfc.io/nubificus/urunc/redis-qemu-unikraft-initrd:latest"
+	containerImage := "harbor.nbfc.io/nubificus/urunc/redis-qemu-unikraft-x86_64:latest"
```

Comment:
If I remember correctly from an internal discussion, it would be useful to keep the initrd somewhere in the image name and not the architecture.